libmamba/tests/src/util/test_graph.cpp

```diff
@@ -339,11 +375,139 @@ TEST_SUITE("graph")
     {
         DiGraph<int> g;
         test_visitor<DiGraph<int>> vis;
-        g.depth_first_search(vis);
+        dfs_raw(g, vis, 0);
```

Comment:
Should we convert `0` to `node_id`? I haven't looked into the details of it to check if the conversion is necessary, just saying because it was done in the declaration of previous used function:
`void depth_first_search(V& visitor, node_id start = node_id(0), bool reverse = false) const;`

Reply:
It does not make a difference (`node_id` is an alias for `std::size_t`) but it's more explicit. I'll add it.